buildspec.yml

```diff
@@ -0,0 +1,33 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - git clone https://github.com/crungehottman/exodus-lambda.git
```

Comment:
Before merge/final review, I will change the repo to https://github.com/release-engineering/exodus-lambda, but my fork and test branch are required for testing.